### ISSUE TITLE
Direction of orientation flipped online vs. locally

### ISSUE THREAD

**Issue by jfkominsky** [psychopy/psychopy#2861]
Confusing behavior more than bug: +angle rotates objects clockwise in PsychoPy but counterclockwise online.

https://discourse.psychopy.org/t/orientation-of-an-imagestim-is-set-differently-in-local-and-online-versions/12902
https://discourse.psychopy.org/t/orientation-of-stimuli-backwards-in-js-vs-python/12961

I'm unclear on whether the builder auto-corrects for this and it's only an issue for code components or if this is just something that's true no matter what, but ideally regardless of whether orientation is being affected via builder or code component direction of rotation should be consistent online vs. off, which may necessitate putting the inversion at a lower and less accessible level of the code in PsychoJS.

**Comment by peircej** [comment]
Yes, most of the world treats orientation the way engineers do (as the angle on a polar coordinate, with zero to the right and anticlockwise being positive). I made the questionable decision with PsychoPy to make it act like a clock or compass. I think PsychoPy has been doing this too long though so I think we should change the JS version to fit my unconventional Python convention!
